src/core/Sampler/Sampler.h

```diff
@@ -108,7 +108,8 @@ class Sampler : public H2Core::Object
 	 * layer will be loaded with a nullptr instead.
 	 */
 	void reinitializePlaybackTrack();
-
+	static float getRatioPan( float fPan_L, float fPan_R );
```

Comment:
Could you use Doxygen-style comments to describe the functions? You can e.g. look at the `OscServer` class methods for reference and build the resulting developer documentation using `build.sh d`.

Reply:
Ok